src/pages/DatasetStatistics.tsx

```diff
@@ -9,15 +9,32 @@ import {find} from 'lodash/fp';
 import {ReadMore} from '../components/ReadMore';
 import {formatDate} from '../libs/utils';
 import {Button} from '@mui/material';
-import {AccessManagement} from '../libs/Models.ts';
+import {History} from 'history';
+import {Dataset, DatasetProperty} from '../types/model';
 
 const LINE = <div style={{borderTop: '1px solid #BABEC1', height: 0}}/>;
 
-export default function DatasetStatistics(props) {
+enum AccessManagement {
+  OPEN = 'open',
+  CONTROLLED = 'controlled',
+  EXTERNAL = 'external'
+}
+
+
+interface DatasetStatisticsProps {
+  history: History,
+  match: {
+    params: {
+      datasetIdentifier: string;
+    }
+  }
+}
+
+export default function DatasetStatistics(props: DatasetStatisticsProps) {
   const {history, match: {params: {datasetIdentifier}}} = props;
-  const [datasetId, setDatasetId] = useState();
-  const [dataset, setDataset] = useState();
-  const [dars, setDars] = useState();
+  const [datasetId, setDatasetId] = useState<number>();
+  const [dataset, setDataset] = useState<Dataset>();
+  const [dars, setDars] = useState<any>();
```

Comment:
Is it possible to type dars as well?

Reply:
It looks like it wouldn't be too hard, although this type doesn't exist yet in duos-ui. The Java type is a List of [DarMetricsSummary](https://github.com/DataBiosphere/consent/blob/develop/src/main/java/org/broadinstitute/consent/http/service/MetricsService.java#L55l)

Reply:
https://broadinstitute.slack.com/archives/C07T2DLHD24/p1734552165921369